test/Strings/unicode_toUpperCase_toLowerCase.js

```diff
@@ -15,6 +16,24 @@ var tests = [
             assert.areEqual("abc\uDC37def", "ABC\uDC37DEF".toLowerCase(), "Invalid unicode should be passed over (mid-string, toLowerCase)");
         }
     },
+    {
+        name: "SpecialCasing.txt",
+        body() {
+            const hasICU = WScript.Platform.INTL_LIBRARY === "icu";
```

Comment:
Just as a FYI, you might also see some failures on really old versions of ICU that have older Unicode data in them.

Reply:
Thats ok. It looks like all of the tests are passing and we only support ICU 55 at the earliest, and even that is hopefully on its way out soon as 16.04 fades compared to 18.04

Reply:
Worst case scenario, since we are getting the CLDR version now from ICU, I could plumb that info through to WScript.Platform in order to do one-off exclusions based on data version.